src/client/array/dc_array.c

```diff
@@ -600,6 +598,7 @@ dc_array_create(tse_task_t *task)
 		D_GOTO(err_put2, rc);
 	}
 
+	tse_task_schedule(open_task, false);
 	tse_task_schedule(update_task, false);
```

Comment:
I tend to think we probably should check pid (or other cookie) to make sure those completion callbacks will only be executed by its own threads. i.e. we only share the tse over different threads.

Reply:
hmm, that seems make tse handling a little complex, and probably with some extreme case that one thread create task and handover to other thread, and itself exit executing.

Reply:
Just discussed with Xuezhao.  we can discuss further, but probably unrelated with this patch.